_neural-search-plugin/index.md

```diff
@@ -21,7 +21,7 @@ In order to ingest vectorized documents, you need to create a Neural Search _pip
 PUT _ingest/pipeline/<pipeline_name>
 ```
 
-In the pipeline request body, The `text_embedding` processor, the only processor supported by Neural Search, converts a document's text to vector embeddings. `text_embedding` uses a `field_map` to know what fields from which to generate vector embeddings and also what field to store the embedding. 
+In the pipeline request body, The `text_embedding` processor, the only processor supported by Neural Search, converts a document's text to vector embeddings. `text_embedding` uses  `field_map`s to determine what fields from which to generate vector embeddings and also which field to store the embedding. 
```

Comment:
"... to determine which fields to generate vector embeddings from and, also, which field to store the embedding in."
Does that retain the meaning?
NOTE: My grade school teacher warned me against dangling prepositions. But they're common now, and I saw some earlier in this doc.